packages/providers/fal/text-to-video/fal_text_to_video/models/kling.py

```
@@ -0,0 +1,95 @@
+
+        return args
+
+    def get_model_info(self) -> Dict[str, Any]:
+        """Get Kling v2.6 Pro model information."""
+        return {
+            **MODEL_INFO.get("kling_2_6_pro", {}),
+            "endpoint": self.endpoint,
+            "pricing": self.pricing
+        }
+
+    def estimate_cost(self, duration: str = "5", generate_audio: bool = True, **kwargs) -> float:
```

Ruff (0.14.10)

Unused method argument: `kwargs`

(ARG002)